robyn/robyn.pyi

```diff
@@ -177,6 +177,8 @@ class QueryParams:
         pass
 
 class Headers:
+    exclude_paths: Optional[list[str]] = None
```

Comment:
Why does the `Headers` class need to have an `exclude_paths` function?

Reply:
`exclude_paths` is not a function, it is a list of string -- to track the excluded paths, to be passed to rust.

Reply:
I am not sure if we should also have this information here. Let me think more